src/hooks/useContract.ts

```diff
@@ -21,7 +21,6 @@ import {
   getIfoPoolContract,
   getPredictionsContract,
   getChainlinkOracleContract,
-  getSouschefV2Contract,
```

Comment:
@chef-jojo it looks like `getSouschefV2Contract` is not used anywhere anymore.

Reply:
export for testing. removed getSouschefV3Contract